src/plugins/julia/README.md

```diff
@@ -0,0 +1,178 @@
+## How to build a parser for Julia
+
+Below are the guiding steps how to build a working parser for the Julia programming language.
+
+### 0. Prerequisite: install Syntax tool
+
+```
+npm install -g syntax-cli
+```
+
+> NOTE: `npm` comes pre-installed with [Node.js](https://nodejs.org/en/)
+
+### 1. Generate a project environment
+
+Use Julia to generate the project by invoking the package management capability, which you can do by hitting the ']' key while in the Julia REPL:
+
+```
+MacBook-Pro-4:Personal Development maestro$ julia
```

Comment:
Let's remove specific computer names ("MacBook-Pro-4:Personal Development maestro"), and just put an abstract command line input